tasks/image-orientation/src/train.py

```diff
@@ -0,0 +1,124 @@
+from typing import Optional
+from pathlib import Path
+
+import logging
+
+from coretex import TaskRun
+from torch.utils.data import DataLoader
+
+import torch
+import torch.nn as nn
+import torch.optim as optim
+
+from .model import OrientationClassifier
+from .utils import calculateAccuracy, EarlyStopping
+
+
+def trainEpoch(
+    trainLoader: DataLoader,
+    model: OrientationClassifier,
+    optimizer: optim.Adam,
+    criterion: nn.MSELoss,
+    device: torch.device
+) -> tuple[float, float]:
+
+    model.train()
+
+    runningTrainLoss = 0.0
+    runningTrainAcc = 0.0
+    for data in trainLoader:
+        inputs = data['image'].to(device)
+        labels = data['label'].to(device)
```

Comment:
There are a couple of places where single quotes are used instead of double quotes